src/main/java/skajihara/projectX/MainContents/Home/repository/AccountRepository.java

```diff
@@ -0,0 +1,12 @@
+package skajihara.projectX.MainContents.Home.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import skajihara.projectX.MainContents.Home.entity.Account;
+
+public interface AccountRepository extends JpaRepository<Account, String> {
```

Comment:
```suggestion
@Repository
public interface AccountRepository extends JpaRepository<Account, String> {
```

Repository アノテーションを付与しないと、DIコンテナからインスタンスがわたってこないかもです。
明示しておいたほうがよい印象があります。

Reply:
アノテーションを付与しました。